app/views/user/swaps/_swap_profile_inner.html.haml

```diff
@@ -1,6 +1,17 @@
 %img.profile-img{ src: other_user.image_url }
 .profile-content
   .profile-name= other_user.redacted_name
+
+  .profile-icons
+    - if other_user.mobile_phone_verified?
+      %i.fa.fa-phone{ title: "Phone number verified" }
+    - if other_user.provider == "twitter"
+      %i.fa.fa-twitter{ title: "Twitter account verified" }
+    - if other_user.provider == "facebook"
+      %i.fa.fa-facebook{ title: "Facebook account verified" }
+    - if other_user.provider == "devise_email"
+      %i.fa.fa-envelope{ title: "Email verified" }
```

Comment:
We don't verify email yet sadly - that's #15.  And with Twitter and Facebook we also now require email.  The difference is currently that with email logins we provide the address to the swap partner.  Once #102 is resolved we can share emails even with social login.

```suggestion
      %i.fa.fa-envelope{ title: "Email address available" }
```